src/main/webapp/scripts/map.js

```diff
@@ -77,10 +71,8 @@ function addMarker(map) {
 /* Place Marker Where Map is Clicked On & Show Popup*/
 function placeMarkerAndPanTo(latLng, map) {
   const greyIcon = {
-    url: "/images/grey-marker.png", // url
-    scaledSize: new google.maps.Size(30, 40), // scaled size
-    origin: new google.maps.Point(0, 0), // origin
-    anchor: new google.maps.Point(0, 0) // anchor 
```

Comment:
unlike `origin`, the default value for `anchor` is probably not `(0, 0)` so you might need to specify it (unless you're happy with the current default value of course!)
https://developers.google.com/maps/documentation/javascript/reference/marker#Icon

Reply:
Thanks - I checked the default value for the anchor and it works better than (0, 0)!